code/modules/mob/living/simple_animal/simple_animal.dm

```diff
@@ -509,7 +510,10 @@
 			alone = 0
 			continue
 	if(alone && partner && children < 3)
-		new childtype(loc)
+		if(prob(5) && rarechildtype)
+			new rarechildtype(loc)
+		else
+			new childtype(loc)
```

Comment:
Except I don't want an equal chance to have the rare one. Rare one needs to be rare.